platforms/x86_64/Windows-Qemu-Test-Setup.md

```diff
@@ -0,0 +1,145 @@
+# Windows Qemu Test Setup
```

Comment:
This instructions are awesome!

Reply:
I needed to write them down, lest I forget how to do it myself.